File: BOJ/C++/1100-하얀칸.cpp
#include <iostream>

int main()
{
    char input[9];
    int count = 0;
    for(int i = 0; i <8; i++)
    {
        std::cin >> input;
        if(i%2)
        {
            for(int k = 1; k<8; k += 2)
            {
                if(input[k] == 'F')
                {
                    count += 1;
                }
            }
        }
        else
        {
            for(int k = 0; k<8; k += 2)
            {
                if(input[k] == 'F')
                {
                    count += 1;
                }
            }
        }
        
    }
    std::cout << count<<'\n';
    return 0;
}

File: BOJ/Python/1100-하얀칸.py
count = 0
for i in range(8):
    a = input()
    if i%2:
        for i in range(1, 8, 2):
            if a[i] == 'F':
                count += 1
    else:
        for i in range(0, 8, 2):
            if a[i] == 'F':
                count += 1
print(count)